### PR TITLE
Language clarifications

### PR DESCRIPTION
**Pull request by EwanC**
Address feedback around the language used in the specification. There should are no intended changes to extension behaviour.

### CODE REVIEW

File: sycl/doc/extensions/proposed/sycl_ext_oneapi_graph.asciidoc [hunk 1]
```diff
@@ -834,6 +842,9 @@ compatible with the proof-of-concept implementation, as the proof-of-concept
 implementation is currently under development.
 ====
 
+Examples for demonstrative purposes only, and may leave out details such as how
```
Comment (Bensuo) [inline sycl/doc/extensions/proposed/sycl_ext_oneapi_graph.asciidoc:881]:
```suggestion
Examples are for demonstrative purposes only, and may leave out details such as how
```

**Review by Bensuo — APPROVED** [review]
LGTM, just one nitpick!

**Review by reble — APPROVED** [review]
one of my recent changes #29 introduced a conflict with this patch. We might want to move the description of dependencies as a property to a separate subsection.

> language-clarifications
* `dep` - Nodes for the created node to be dependent on, in addition to the
  dependent nodes identified from the `cgf` requisites.
=======
* `propList` - Zero or more properties can be provided to the constructed node 
  via an instance of `property_list`. 
> sycl-graph-update

* `depends_on` - Property of nodes for the created node to be dependent on, in addition to the
  dependent nodes identified from the `cgf` requisites.

**Comment by EwanC** [comment]
> * Property of nodes for the created node to be dependent on, in addition to the
>       dependent nodes identified from the `cgf` requisites.

Thanks for pointing out the conflict, I've resolved that and pushed a [new commit](https://github.com/reble/llvm/pull/35/commits/548c4148714b0835c463f88c447f0fd8e9f2cff6) for describing the `depends_on` property in it's own subsection

**Comment by EwanC** [comment]
Merging as this has 2 approvals, and i've addressed the comments

**Pull request merged**